From: Philip Hands <phil@hands.com>
Date: Sun, 7 Mar 2021 07:08:05 +0100
Subject: stop upsetting pod2man

---
 script/client         | 9 +++++++++
 script/dump_templates | 4 ++++
 2 files changed, 13 insertions(+)

diff --git a/script/client b/script/client
index 1bae6f5..83f642b 100755
--- a/script/client
+++ b/script/client
@@ -92,8 +92,12 @@ jobs/1/artefact post --form file.file=bar file.filename=bar.log
 
 print help
 
+=back
+
 =head2 Archive mode
 
+=over 4
+
 =item B<--archive, -a> DIRECTORY
 
 Archive mode: Download assets and test results from a job to DIRECTORY.
@@ -107,6 +111,8 @@ Archive mode: Include thumbnails
 Archive mode: Download assets that do not exceed the specified limit in bytes
 The default limit is 200 MB.
 
+=back
+
 =head1 SYNOPSIS
 
 Interact with the openQA API by specified route entry points and optionally
@@ -116,6 +122,8 @@ for available API routes.
 
 Common top level entry points: jobs, workers, isos.
 
+=over 4
+
 =item client --host openqa.example.com jobs
 
 List all jobs. Caution: this will take a very long time or even timeout on big
@@ -147,6 +155,7 @@ Trigger jobs on iso B<bar.iso> matching test suite B<blah>.
 
 Download all assets and test logs and images from job B<42> with asset limit of B<1GB> to B</path/to/directory>.
 
+=back
 =cut
 
 use strict;
diff --git a/script/dump_templates b/script/dump_templates
index 026f0c2..4ce37af 100755
--- a/script/dump_templates
+++ b/script/dump_templates
@@ -89,6 +89,8 @@ print help
 
 =head1 SYNOPSIS
 
+=over 4
+
 =item dump_templates --host openqa.example.com Products
 
 =item dump_templates --host openqa.example.com --group "openSUSE Tumbleweed" --full
@@ -97,6 +99,8 @@ print help
 
 =item dump_templates --host openqa.example.com --machine "64bit"
 
+=back
+
 =head1 DESCRIPTION
 
 Dump openQA database tables for Machines TestSuites Products
