From: Philip Hands <phil@hands.com>
Date: Sat, 13 Mar 2021 06:50:27 +0100
Subject: fetchneedles -h: mention the needles repo, if set

---
 script/fetchneedles | 18 +++++++++++++-----
 1 file changed, 13 insertions(+), 5 deletions(-)

diff --git a/script/fetchneedles b/script/fetchneedles
index 2ceac47..1771908 100755
--- a/script/fetchneedles
+++ b/script/fetchneedles
@@ -1,8 +1,7 @@
 #!/bin/sh -e
-[ "$1" = "-h" ] || [ "$1" = "--help" ] && echo "Get openSUSE tests and needles from https://github.com/os-autoinst/os-autoinst-distri-opensuse" && exit
-
 : "${dbuser:="geekotest"}"
 
+: "${dist_name:=${dist:-"OpenSUSE"}}" # the display name, for the help message
 : "${dist:="opensuse"}"
 : "${giturl:="git://github.com/os-autoinst/os-autoinst-distri-opensuse.git"}"
 : "${branch:="master"}"
@@ -10,13 +9,22 @@
 : "${username:="openQA web UI"}"
 : "${product:="$dist"}"
 
-: "${needles_separate:="1"}"
-: "${needles_giturl:="git://github.com/os-autoinst/os-autoinst-needles-opensuse.git"}"
-: "${needles_branch:="master"}"
+: "${needles_separate:="0"}"
+: "${needles_giturl:="https://salsa.debian.org/qa/openqa/openqa-needles-debian.git"}"
+: "${needles_branch:="debian-needles"}"
 
 : "${updateall:="0"}"
 : "${force:="0"}"
 
+if [ "$1" = "-h" ] || [ "$1" = "--help" ] ; then
+    echo "Get ${dist_name} tests and needles from:\n\t\t${giturl}"
+    if [ 1 = ${needles_separate} ] ; then
+        echo "\tand\t${needles_giturl}"
+    fi
+    echo "\n(environment variables mentioned in '$0' can be set to change these values)"
+    exit
+fi
+
 dir="/var/lib/openqa/share/tests"
 if [ -w / ]; then
     if [ ! -e "$dir/$dist" ]; then
