From: Philip Hands <phil@hands.com>
Date: Sat, 13 Mar 2021 16:44:58 +0100
Subject: Debian-ify fetchneedles

---
 script/fetchneedles | 16 ++++++++++------
 1 file changed, 10 insertions(+), 6 deletions(-)

diff --git a/script/fetchneedles b/script/fetchneedles
index 1771908..9e584c7 100755
--- a/script/fetchneedles
+++ b/script/fetchneedles
@@ -1,14 +1,15 @@
 #!/bin/sh -e
 : "${dbuser:="geekotest"}"
 
-: "${dist_name:=${dist:-"OpenSUSE"}}" # the display name, for the help message
-: "${dist:="opensuse"}"
-: "${giturl:="git://github.com/os-autoinst/os-autoinst-distri-opensuse.git"}"
-: "${branch:="master"}"
-: "${email:="openqa@$HOST"}"
+: "${dist_name:=${dist:-"Debian"}}" # the display name, for the help message
+: "${dist:="debian"}"
+: "${giturl:="https://salsa.debian.org/qa/openqa/openqa-tests-debian.git"}"
+: "${branch:="debian"}"
+: "${email:="openqa@$(hostname -f)"}"
 : "${username:="openQA web UI"}"
 : "${product:="$dist"}"
 
+: "${git_lfs:="1"}"
 : "${needles_separate:="0"}"
 : "${needles_giturl:="https://salsa.debian.org/qa/openqa/openqa-needles-debian.git"}"
 : "${needles_branch:="debian-needles"}"
@@ -29,7 +30,7 @@ dir="/var/lib/openqa/share/tests"
 if [ -w / ]; then
     if [ ! -e "$dir/$dist" ]; then
         mkdir -p "$dir/$dist"
-        chown $dbuser:www "$dir/$dist"
+        chown $dbuser:www-data "$dir/$dist"
     fi
     echo "running as root, re-exec as $dbuser ..."
     exec sudo -u $dbuser env \
@@ -123,6 +124,9 @@ else
             git config user.email "$email"
             git config user.name "$username"
         fi
+        if [ "$git_lfs" = 1 ]; then
+            git lfs install --local
+        fi
     else
         do_fetch "$target"
     fi
